### PR TITLE
Adds three new rare dice to the game

### PR DESCRIPTION
**Pull request by Incoming5643**
the 8bd20 (Eight Ball D20)
d10_icon

the 4dd6 (Hyper D6)
d10_icon

the space cube (It's just a D6)
d10_icon

### CODE REVIEW

**Comment by TehZombehz** [comment]
Magic missile!

**Comment by Supermichael777** [comment]
quiet of 6e will use hyper 6

File: code/game/objects/items/weapons/dice.dm [hunk 1]
```diff
@@ -59,6 +65,16 @@
 /obj/item/weapon/dice/d6
 	name = "d6"
 
+/obj/item/weapon/dice/d6/space
+	name = "space cube"
+	desc = "A die with six sides. 6 TIMES 255 TIMES 255 TILE TOTAL EXISTENCE, SQUARE YOUR MIND OF EDUCATED STUPID: 2 DOES NOT EXIST."
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/dice.dm]: TIME
CUBE

**Review by Iamgoofball — APPROVED** [review]

**Comment by PKPenguin321** [comment]
Space cube o7

File: code/game/objects/items/weapons/dice.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/objects/items/weapons/dice.dm]: needs a magic version that teleports you randomly

**Comment by optimumtact** [comment]
When the Sun shines upon Earth, 2 – major Time points are created on opposite sides of Earth – known as Midday and Midnight. Where the 2 major Time forces join, synergy creates 2 new minor Time points we recognize as Sunup and Sundown. The 4-equidistant Time points can be considered as Time Square imprinted upon the circle of Earth. In a single rotation of the Earth sphere, each Time corner point rotates through the other 3-corner Time points, thus creating 16 corners, 96 hours and 4-simultaneous 24-hour Days within a single rotation of Earth – equated to a Higher Order of Life Time Cube.

**Pull request merged**